__tests__/layer/fully-connected.js

```diff
@@ -1,14 +1,15 @@
-import gpuMock from 'gpu-mock.js';
-import {
-  predict,
-  predict3D,
-  compareBiases,
-  compareFilterDeltas,
-  compareFilterDeltas3D,
-  compareInputDeltas,
-  compareInputDeltas3D,
-} from '../../src/layer/fully-connected';
-import { onePlusPlus2D, zero2D } from '../test-utils';
+const gpuMock = require('gpu-mock.js');
+const fc = require('../../src/layer/fully-connected');
```

Comment:
You can use Object Destructuring here to simplify requires. like this: 
```js
const { predict, predict3D, compareBiases, compareFilterDeltas, compareFilterDeltas3D, compareInputDeltas, compareInputDeltas3D } = require('../../src/layer/fully-connected');
```

Reply:
Got it, that would make it tighter.  Should I put that in a new commit and submit a new PR for that?  Or is there a way to append it to this PR?

Reply:
Just push more commits to this branch, these will be automatically added to this pull request. 👍